.circleci/config.yml

```diff
@@ -20,4 +17,7 @@ workflows:
   version: 2
   build_and_deploy:
     jobs:
-      - deploy-website
+      - deploy-website:
```

Comment:
See https://discuss.circleci.com/t/triggering-build-on-changes-to-only-one-branch-when-using-a-workflow/20430/3